packages/circuits/crates/libs/greco/src/lib.nr

```diff
@@ -233,27 +233,27 @@ impl<let N: u32, let L: u32> Greco<N, L> {
     ///
```

Comment:
is the `SIZE` comment still valuable and correct? I'm asking since we removed the `(10 * N - 4) * L + 4 * N` from the payload and we are using the length of the input

Reply:
Deleted those comments in the latest commit. Thanks for spotting them!